aten/src/ATen/native/native_functions.yaml

```diff
@@ -1600,6 +1600,11 @@
 
 - func: celu_(Tensor(a!) self, Scalar alpha=1.0) -> Tensor(a!)
 
+- func: mkldnn_sigmoid(Tensor self) -> Tensor
+  dispatch:
+    MkldnnCPU: mkldnn_sigmoid
+  requires_tensor: True
+
 - func: sigmoid(Tensor self) -> Tensor
   variants: function, method
 
```

Comment:
we probably want to make in-place sigmoid below work too (`sigmoid_(Tensor(a!) self) -> Tensor(a!)`)

Reply:
currently in-place doesn't work in mkldnn tensors